invoke.yaml

```diff
@@ -66,6 +66,7 @@ addresses:
     nb_threads: # number of threads to use
 #   nb_shards:  # control the number of shards of the ES index
 #   nb_replicas:  # control the number of replicas of the ES index
+  osm: true
```

Comment:
Maybe we want to be able to use an OSM file different from the one used to import roads (and other stuff?).

For example: I think our first intention is to only import addresses from Europe, but not affecting other parts of the pipeline.

Reply:
That's a good point. Or should we implement a filter for osm data before running this importer ?

Reply:
It was actually for testing that I made this change. But I think we could keep it minus adding a comment on about what it's used for.

Reply:
Maybe we should also add the possibility of specifying an URL, in case we need to use a separate OSM file? We would cover three cases:

 - `false`: we don't import any OSM address
 - `true`: import OSM addresses from `osm.url`
 - `"http://...pbf"`: import OSM addresses from a "custom" URL

Reply:
I don't really like argument that can have so many kind of parameters. So having a fully explicit one might just be better, like for the others with `file:` etc...